contribs/gmf/apps/desktop_alt/js/controller.js

```diff
@@ -26,6 +28,8 @@ app.module.constant('ngeoExportFeatureFormats', [
   ngeo.FeatureHelper.FormatType.GPX
 ]);
 
+// Filter to apply by default on all coordinates.
+app.module.constant('ngeoPointfilter', 'ngeoNumberCoordinates:0:{x} E, {y} N');
```

Comment:
Yes, it can be. But it will be complicated to understand for a GMF user that this value can be overrided/adapted. What should I do ?